rfc/002-fault-tolerance/README.md

```diff
@@ -1,8 +1,8 @@
 # Fault Tolerance
 
-- **Status**: In-Review
+- **Status**: Accepted
 - **Created**: Sep 5, 2022
-- **ETA**: Nov 30, 2022
+- **Accepted**: Nov :8, 2022
```

Comment:
Why is there a colon in there?

Reply:
I have no explanation for this. Fix incoming.

Reply:
#2741